wazo_confgend/confgen.py

```diff
@@ -110,6 +133,6 @@ def _encode_and_cache(self, cache_key, content):
         if not content:
             return
 
-        encoded_content = content.encode('utf-8')
+        encoded_content = content
```

Comment:
Does it not need to be encoded? If not, maybe don't re-assign it and just use the `content` variable